pkg/controller/constraint/constraint_controller.go

```diff
@@ -59,47 +60,15 @@ import (
 )
 
 var (
-	log          = logf.Log.V(logging.DebugLevel).WithName("controller").WithValues(logging.Process, "constraint_controller")
-	discoveryErr *apiutil.ErrResourceDiscoveryFailed
+	log                 = logf.Log.V(logging.DebugLevel).WithName("controller").WithValues(logging.Process, "constraint_controller")
+	discoveryErr        *apiutil.ErrResourceDiscoveryFailed
+	DefaultGenerateVAPB = flag.Bool("generate-vapbinding", false, "control VAPBinding resource generation. Allowed values are false: do not generate VAPBinding for constraint by default, true: generate VAPBinding for constraint by default.")
```

Comment:
I think we can probably name and document this flag better given we know its eventual use... that its mainly there for customizing the binding generation behavior when users are not using scoped EA. "by default" doesn't really carry any of that nuance.

Reply:
Naming things is hard. I am open to naming and description suggestions 😄

Reply:
How about, `generate-vap-binding-without-scoped-actions`?

Reply:
or `create-vap-binding-for-constraints`?